config/sync/block.block.webtools_etrans.yml

```diff
@@ -25,8 +25,14 @@ settings:
   include: ''
   exclude: ''
 visibility:
-  request_path:
-    id: request_path
-    pages: "/node/*/edit\r\n/node/*/delete\r\n/rdf_entity/*/edit\r\n/rdf_entity/*/delete\r\n/taxonomy_term/*/edit\r\n/taxonomy_term/*/delete"
+  request_route:
+    id: request_route
+    routes:
+      0: entity.node.delete_form
+      2: entity.node.edit_form
+      4: entity.rdf_entity.delete_form
+      6: entity.rdf_entity.edit_form
+      8: entity.taxonomy_term.edit_form
+      10: entity.taxonomy_term.delete_form
```

Comment:
This looks like a bug in the route condition module. My guess is that, when exploding the blob, an `array_values()` is missing.